packages/cli/src/lib/bundler/transforms.ts

```diff
@@ -32,6 +32,32 @@ export const transforms = (options: TransformOptions): Transforms => {
 
   const extraTransforms = isDev ? ['react-hot-loader'] : [];
 
+  // Make sure that imported css styles are included before style
+  // elements created by makeStyle to keep a stable insertion
+  // order which results in a stable evaluation order of the css
+  // selectors.
+  //
+  // insert is executed in the browser without transformation,
+  // avoid using new JS features here!
+  function insertElementAfterTitle(element: any) {
+    const head = document.head;
+    // An alternative is to include an explicit insertion point
+    // like <noscript id="css-insertion-point"></noscript> in the
+    // index.html, and retrieve it here like:
+    // var insertionPoint = document.getElementById(
+    //   'css-insertion-point',
+    // );
+    // But that would be a breaking change, therefore we start
+    // with abusing the <title> tag.
+    const insertionPoint = document.querySelector('title');
```

Comment:
Should this be looking ONLY for direct children of `head` instead? Just to be safe.

Reply:
Hope that there are no other title elements, but we could change the code to make sure :D

Reply:
Done 🙂